services/hanke-service/src/main/kotlin/fi/hel/haitaton/hanke/domain/HankeYhteystieto.kt

```diff
@@ -5,23 +5,23 @@ import java.time.ZonedDateTime
 
 // e.g. omistaja, arvioija, toteuttaja
 data class HankeYhteystieto(
```

Comment:
Should we add toLogString() that makes sure the personal information is not printed for this domain class?